features/user/managing_customers/adding_customer.feature

```diff
@@ -0,0 +1,30 @@
+@managing_customers
+Feature: Adding a new customer
+    In order to connects orders with client
+    As an Administrator
+    I want to add a customer to the store
+
+    Background:
+      Given I am logged in as an administrator
+
+    @ui
+    Scenario: Adding a new customer with required fields
+        Given I want to create a new customer
+        When I specify its first name as "Luke"
+        And I specify its last name as "Skywalker"
+        And I specify its email as "l.skywalker@gmail.com"
+        And I add it
+        Then I should be notified that it has been successfully created
+        And the customer "l.skywalker@gmail.com" should appear in the store
+
+    @ui
+    Scenario: Adding a new customer with gender and birthday
+        Given I want to create a new customer
+        When I specify its first name as "Luke"
+        And I specify its last name as "Skywalker"
+        And I specify its email as "l.skywalker@gmail.com"
+        And I specify its birthday as "1892-01-03"
+        And I select "Male" from gender
+        And I add it
+        Then I should be notified that it has been successfully created
+        And the customer "l.skywalker@gmail.com" with this birthday and this gender should appear in the store
```

Comment:
We have resigned with explicit checks on index page. Last step from previous scenario would be good.